mesh/mesh-io-tcpserver.c

```diff
@@ -61,24 +63,29 @@ struct mesh_io_private {
 	mesh_io_ready_func_t ready_callback;
 
 	struct l_io		*server_io;
-	struct l_queue		*client_io;
+	struct l_queue		*conns;
 
 	struct l_timeout	*tx_timeout;
 	struct l_queue		*rx_regs;
 	struct l_queue		*tx_pkts;
 
 	/* Simple filtering on AD type only */
-	uint8_t				filters[4];
+	uint8_t			filters[4];
 	struct tx_pkt		*tx;
 
 	/* TLS context */
-	SSL_CTX				*tls_ctx;
+	SSL_CTX			*tls_ctx;
 
-	uint8_t				*uuid;
-	uint8_t				*dev_key;
-	uint8_t				*net_key;
+	const char		*dbus_path;
+	struct l_dbus		*dbus;
+	struct tcpserver_acl	*acl;
 };
 
+struct tcpserver_conn {
+	uint8_t			identity_buf[IDENTITY_BUF_LEN];
```

Comment:
I'd call it `identity_str`

Reply:
`identity` here is kept in the binary representation.